src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import java.util.LinkedList;
+
+import duke.task.Task;
+
+/**
+ * A class to handle operations on the task list.
+ * A class that abstracts all operations regarding the task list, so that outer classes
+ * do not need to operate on a <code>LinkedList</code> anymore.
+ */
+public class TaskList {
+    private LinkedList<Task> list;
```

Comment:
I am not sure if LinkedList is the best implementation for tasklist. I think that ArrayList is better as there are alot of index calls which would run in O(1) time.